api-reference/docstubs/api/group-post-members.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Create members"
```

Comment:
Looks like we'd mixed up how we treat navigations and contained navigations.
Per spec:
For navigations, we should have Add, List and Remove
For contained navigations, we should treat like entitysets, and have Create, Get, List, Update and Delete 

So here:
title:  Add members
description: Add members to a group by posting to the members collection
Add members to a [group](../resources/group.md)  by posting to the **members** collection